.jenkins/pytorch/build-mobile.sh

```diff
@@ -10,12 +10,13 @@ COMPACT_JOB_NAME="${BUILD_ENVIRONMENT}"
 
 source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
 
-echo "Clang version:"
-clang --version
+# Install torch & torchvision - used to download & trace test model.
+pip install torch torchvision --progress-bar off
```

Comment:
I see iOS simulator test is doing this - I'm not entirely sure whether it's a good idea to download and install torch & torchvision on every PR.

My goal is to download & trace & run a non-trivial model so I chose MobileNetV2 - shall I create a dummy TorchScript model with similar set of ops like MobileNetV2 but with small dummy weights instead so that I can commit the test model without downloading it from network?
Are there any similar integration tests for non-mobile builds that I can reuse?

Reply:
It's not *great* (I commented on this at https://github.com/pytorch/pytorch/pull/30133#issuecomment-557130164) but it's certainly the most convenient way to dump the model (and we do this in a few other places). An alternative is to save the model in s3 directly and then suck it down directly, but that makes it more difficult to update the model later.

Reply:
As for your other question, let me read the patch and get back to you on it.

Reply:
> It's not _great_ (I commented on this at [#30133 (comment)](https://github.com/pytorch/pytorch/pull/30133#issuecomment-557130164)) but it's certainly the most convenient way to dump the model (and we do this in a few other places). An alternative is to save the model in s3 directly and then suck it down directly, but that makes it more difficult to update the model later.

I saw you commented there - "concerned about relying on nightlies for per PR tests" - it installs stable torch, isn't it?